### PR TITLE
update test

### PR DESCRIPTION
**Pull request by SimonLab**
update tests to have 100% coverage, see https://github.com/akitten/public-profile-page-url-finder/issues/13

### CODE REVIEW

File: test/e2e.test.js [hunk 1]
```diff
@@ -36,12 +36,23 @@ test('Check that any-na linkedIn links are not selected', function(t) {
 })
 
 test('Check that the link of a pdf are correctly parsed', function(t) {
-  var keywords =  [ 'Abdi', 'Ahmed', 'Founders', 'and', 'Coders'];
+  var keywords =  [ 'Ines', 'Teles', 'Founders', 'Coders', 'pdf'];
```
Comment (nelsonic) [inline test/e2e.test.js]: Where are you getting the pdfs from?

**Comment by nelsonic** [comment]
Does this need to be a new version on NPM? would you mind increasing the version in `package.json` ?

**Comment by nelsonic** [comment]
The badges need to be updated to work... do you have time to do this? (2mins)
and include the commit in this PR?

File: test/e2e.test.js (hunk 1 above)
Comment (SimonLab) [inline test/e2e.test.js]: I just had the pdf in the keywords to get some random pdf links in the google results.

pdflinks

**Comment by SimonLab** [comment]
new badges link and version increased :white_check_mark:

**Comment by nelsonic** [comment]
sweet. :lollipop:

**Comment by nelsonic** [comment]
@SimonLab new version on NPM: `1.0.8` :+1:

**Pull request merged**